packages/amplify-graphql-auth-transformer/src/resolvers/helpers.ts

```diff
@@ -251,6 +252,48 @@ export const generateOwnerClaimExpression = (ownerClaim: string, refName: string
   return compoundExpression(expressions);
 };
 
+/**
+ * Sets the value of owner field if the user is already Authorized
+ */
+export const populateOwnerField = (
```

Comment:
I think that this field should be something like `generatePopulateOwnerField` so that it's clear that it's generating VTL code.